label_studio/core/views.py

```diff
@@ -169,6 +170,21 @@ def samples_paragraphs(request):
     return HttpResponse(json.dumps(result), content_type='application/json')
 
 
+def heidi_tips(request):
+    """Fetch live tips from github raw liveContent.json to avoid caching and client side CORS issues"""
+    url = 'https://raw.githubusercontent.com/HumanSignal/label-studio/refs/heads/develop/web/apps/labelstudio/src/components/HeidiTips/liveContent.json'
```

Comment:
are we sure that this domain have wide open CORS policy? asking just in case

Reply:
There is no CORS in backend code, CORS is a browser security policy to protect the user. This is why it must be proxied from the backend, otherwise this would only fail CORS in the browser all the time.